### ISSUE TITLE
Too many open files error

### ISSUE THREAD

**Issue by curquiza** [meilisearch/MeiliSearch#1736]
When running the SDK tests, we get "too many open files" error

Steps

- Run MeiliSearch with a master key:
```
cargo run --release -- --master-key=masterKey
```

- Clone the meilisearch-js repo and run the tests
```
git clone git@github.com:meilisearch/meilisearch-js.git
cd meilisearch-js
yarn
yarn test
```

On macOS
The bug was not present in v0.22.0

**Comment by curquiza** [comment]
Some investigation: doing the same test with MeiliSearch in docker does not fail

Can someone do the test on Linux?

**Comment by MarinPostma** [comment]
I have the same issue on linux, I set `ulimit -Sn 10000`, and it works.

**Comment by ManyTheFish** [comment]
Hello @curquiza, I don't manage to reproduce the failure, I'm trying it on` Meilisearch/main`, `meilisearch-js/main`, and `node v16.10.0`.

```
Test Suites: 40 passed, 40 total
Tests:       1400 passed, 1400 total
```

**Comment by curquiza** [comment]
Closed by #1783
